### PR TITLE
localize issue template for ko

### PR DESCRIPTION
**Pull request by SnowMarble**
#8918  for ko

### CODE REVIEW

**Comment by SnowMarble** [comment]
@mdn/core-dev
I think it needs core dev's review.

**Review by caugner — APPROVED** [review]
Code changes looks good, please ping me once the content itself has been reviewed.

**Comment by yechoi42** [comment]
@hochan222 님 이 이슈 대신 확인 부탁드려요!

File: .github/ISSUE_TEMPLATE/page-report-ko.yml [hunk 1]
```diff
@@ -1,62 +1,62 @@
...
       value: |
         ---
-        You're finished! The following fields are prefilled. Please click **Submit new issue**.
+        모두 끝났어요! 아래 필드는 미리 채워져 있으니 **Submit new issue**를 클릭해주세요.
   - type: textarea
     id: metadata
     attributes:
       label: MDN metadata
-      description: Set automatically. Do not modify.
+      description: 자동으로 설정되니 수정하지마세요.
```
Comment (hochan222) [inline .github/ISSUE_TEMPLATE/page-report-ko.yml:62]:
```suggestion
      description: 자동으로 설정되니 수정하지 마세요.
```

**Review by hochan222 — CHANGES_REQUESTED** [review]
LGTM. 고생 많으셨습니다 :)

아래 한 가지 사항 반영 뒤 caugner님 멘션 드리면 될듯합니다!

**Comment by SnowMarble** [comment]
@caugner local reviewer checked it out, and now it looks like everything is ready. 💪

**Comment by caugner** [comment]
@hochan222 If @SnowMarble has addressed all your requested changes, please approve the changes, and I'll merge. 🙂

**Review by caugner — APPROVED** [review]

**Review by hochan222 — APPROVED** [review]
Sorry everyone! I didn't click approve. 

There are no more content-related issues left. No problem!

**Pull request merged**